widgetastic_manageiq/__init__.py

```diff
@@ -61,6 +61,58 @@
 from cfme.exceptions import ItemNotFound
 
 
+class SnapshotMemorySwitch(Widget):
+    """ This replaces the old-style checkbox when working with snapshot memory.
+
+    At first glance on the page it looks like a BootstrapSwitch but it really isn't.
+    Basic switch controls are implemented here along with fill and read methods,
+    so this should be usable as any other widget.
+    """
+
+    LOCATOR = u'//div[contains(@class, "bootstrap-switch-snap_memory")]'
```

Comment:
You should use `ROOT` here, and remove your `__locator__` property. Widgetastic will create it automatically when `ROOT` is used.